### DISCUSSION TITLE
How can I make a hiding taskbar?

### DISCUSSION THREAD

**Discussion by VV-SH** [valinet/ExplorerPatcher#365] [Q&A]
Subj
In the previous version I did, and at 22000.318.37.2 I could not find. :(

**Comment by VV-SH** [comment]
Have already found...

**Reply by valinet** [reply]
It's useful to describe the steps you took to fix it, so that other users facing the same problem can take your advice and avoid opening a duplicate issue. It's about sharing back as well. Thanks.

**Comment by VV-SH** [comment]
![16-11-2021_10-54-15](https://user-images.githubusercontent.com/73989146/141944382-613ecf90-f379-4837-807d-497f74f01975.png)
